docs/nb/Simple_Scripts.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Simple Scripts with SpecDB (v1.2.1)"
+    "# Simple Scripts with SpecDB (v1.3)"
```

Comment:
Also, in the the usage line of the specdb_plot script in this notebook needs to be updated:
`-s SURVEY` -> `-g GROUP`